docs/general/kusama/kusama-statemine-ledger.md

```diff
@@ -103,9 +103,9 @@ prompt your ledger for confirmation when you initiate a transaction.
 
 ### Working on both Kusama and Statemine
 
-Polkadot and Kusama accounts will show up in Statemint and Statemine respectively, even if they're not set to be used on all chains. What this means is that if you already have a Kusama Ledger account, that account will also be available on Statemine. But the opposite is not the case: an account created specifically on Statemine or Statemint won't show up on Kusama or Polkadot respectively. 
+Polkadot and Kusama accounts from the [extension](https://wiki.polkadot.network/docs/learn-account-generation#polkadotjs-browser-extension) will show up in Statemint and Statemine respectively, even if they're not set to be used on all chains. The reasoning behind this is that Statemine, which is a common good parachain on Kusama uses KSM as its native token. Hence, the same accounts that were selected for Kusama will also be injected into Statemine. What this means is that if you already have a Kusama Ledger account configured on the extension, that account will also be available on Statemine. But the opposite is not the case: an account created specifically on Statemine or Statemint won't show up on Kusama or Polkadot respectively. 
```

Comment:
The additions that this only applies to extension accounts is incorrect. All accounts, regardless of type, will be available on Statemine and Statemint.

Also, this: 
`Statemine, which is a common good parachain on Kusama uses KSM as its native token. Hence, the same accounts that were selected for Kusama will also be injected into Statemine.`
should probably be:
`Statemint and Statemine, which are common good parachains on Polkadot and Kusama respectively, use DOT and KSM as their native tokens. Hence, accounts on Polkadot and Kusama makes sense to be also available on their respective common good parachains`

Reply:
perfect!